app/views/courses/TeacherClassView.coffee

```diff
@@ -332,19 +333,21 @@ module.exports = class TeacherClassView extends RootView
     # TODO: Does not yield .csv download on Safari, and instead opens a new tab with the .csv contents
     window.tracker?.trackEvent 'Teachers Class Export CSV', category: 'Teachers', classroomID: @classroom.id, ['Mixpanel']
     courseLabels = ""
-    courseOrder = []
-    courses = (@courses.get(c._id) for c in @classroom.get('courses'))
-    courseLabelsArray = helper.courseLabelsArray courses
+    courses = (@courses.get(c._id) for c in @sortedCourses)
+    courseLabelsArray = helper.courseLabelsArray(courses)
```

Comment:
I'm not a fan of calculating things like `@sortedCourses` just once in `onLoaded`, since if we ever add something that fetches the classroom again, this button will produce incorrect results based on whatever was loaded initially, and it would fail quietly.

There's added weirdness here from having trimCourses (in classroom) and proper Courses both stored separately, but I would prefer to simply sort the courses every time you click the button. It's not a CPU-costly operation, so it's better to do it the way that's more future-bug-proof.